Blockzilla/OverlayView.swift

```diff
@@ -9,24 +10,30 @@ protocol OverlayViewDelegate: class {
     func overlayViewDidTouchEmptyArea(_ overlayView: OverlayView)
     func overlayViewDidPressSettings(_ overlayView: OverlayView)
     func overlayView(_ overlayView: OverlayView, didSearchForQuery query: String)
+    func overlayView(_ overlayView: OverlayView, didSubmitText text: String)
 }
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-
+    //Settings Button
     fileprivate let settingsButton = UIButton()
-
+    //Search Button
     private let searchButton = InsetButton()
     private let searchBorder = UIView()
     private var bottomConstraint: Constraint!
     private var presented = false
     private var searchQuery = ""
+    
+    //Copy Button
+    private let copyButton = InsetButton()
+    private let copyBorder = UIView()
+
 
     init() {
         super.init(frame: CGRect.zero)
-
         KeyboardHelper.defaultHelper.addDelegate(delegate: self)
-
+        
+        //Search Button
```

Comment:
I don't think any of these changes are necessary given my feelings about comments and minimal PRs. Let me know if you feel strongly otherwise and we can discuss.